releasenotes/notes/moved-priority-constants-316d6b3b791d6b3e.yaml

```diff
@@ -0,0 +1,4 @@
+---
+deprecations:
+  - |
+    Moved `ddtrace.ext.priority` constants into `ddtrace.constants` module.
```

Comment:
```suggestion
    Moved ``ddtrace.ext.priority`` constants into ``ddtrace.constants`` module.
```

Reply:
We might also want to add an `upgrade:` note here as well, like `instead of using ddtrace.ext.priority.AUTO_KEEP use ddtrace.constants.AUTO_KEEP.... ` type message ?

Reply:
I'll add an upgrade section to the release notes!